src/store.js

```diff
@@ -15,7 +15,7 @@ const readSettings = () => {
 const useSettings = create((update, read) => ({
   ...readSettings(),
 
-  get: (key) => at(read(), key),
+  get: (key) => at(read(), key)[0],
```

Comment:
lodash's [`get()`](https://lodash.com/docs#get) could be used

Reply:
good catch! that's not the first time I miss `get()` hehe